src/sections/SideBar/components/IconMaker/IconMaker.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { HomeOutlinedIcon, 
+  PersonOutlineOutlinedIcon, 
+  MailOutlineOutlinedIcon, 
+  NotesOutlinedIcon, 
+  FacebookIcon, 
+  InstagramIcon, 
+  LinkedInIcon, 
+  GitHubIcon, icons } from "./index";
+
+const IconMaker = ({ icon, iconColor, mouseOver, mouseOut }) => {
+  const selectIcon = (icon) => {
```

Comment:
no if else conditional rendering just 
```

{condition && element}
{condition && element}
{condition && element}
{condition && element}

```

Reply:
like

Reply:
`{condition && <element1> </element1>}`

Reply:
Also please define icons in a constants js file at root level so that it can be used by all sections or components. Please make it Mayus like ICONS